src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,74 @@
+package duke;
+
+import duke.command.Command;
+import duke.exception.DukeException;
+import duke.exception.InvalidInputException;
+import duke.exception.StorageFileException;
+import duke.parser.Parser;
+import duke.storage.CommandHistory;
+import duke.storage.Storage;
+import duke.task.TaskList;
+import duke.ui.Ui;
+
+/**
+ * The main Duke class to run
+ */
+public class Duke {
+    private final Storage storage;

```

Comment:
Perhaps the storage variable name could be in caps because it is final. Similarly for other final variables.